js/main.js

```diff
@@ -15,14 +15,17 @@ const $inputTitle = document.querySelector('#title');
 
 function formHandler(event) {
   event.preventDefault();
+  const $textArea = document.querySelector('#notes');
```

Comment:
You shouldn't be doing `querySelector`s inside functions. By having it inside of here, you're forcing us to traverse the DOM over and over again every single time this function gets called. If you just do it once globally, we don't need to search again